### PR TITLE
Fix a bug if there are no stories in a sketch and add properties to View

### PR DESCRIPTION
**Pull request by kiddinn**
Fixes a bug when trying to call `list_stories` on a sketch with no stories in it.

Also adding few more properties of a view object.

### CODE REVIEW

File: api_client/python/timesketch_api_client/sketch.py [hunk 1]
```diff
@@ -344,7 +344,13 @@ def list_stories(self):
             self.api.api_root, self.id)
         response = self.api.session.get(resource_url)
         response_json = response.json()
-        stories = response_json.get('objects', [[]])[0]
+        objects = response_json.get('objects')
```
Comment (berggren) [inline api_client/python/timesketch_api_client/sketch.py]: maybe story_objects ?

File: api_client/python/timesketch_api_client/view.py [hunk 2]
```diff
@@ -39,15 +39,58 @@ def __init__(self, view_id, view_name, sketch_id, api):
...
+            name: String with the attribute name.
+            default_value: The default value if the attribute does not exit,
+                defaults to None.
+            refresh: If set to True then the data will be refreshed.
+
+        Returns:
+            The dict value of the key "name".
+        """
+        view = self.lazyload_data(refresh_cache=refresh)
+        objects = view.get('objects')
```
Comment (berggren) [inline api_client/python/timesketch_api_client/view.py]: view_objects?

File: api_client/python/timesketch_api_client/sketch.py (hunk 1 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/sketch.py]: done

File: api_client/python/timesketch_api_client/view.py (hunk 2 above)
Comment (kiddinn) [reply api_client/python/timesketch_api_client/view.py]: done

**Pull request merged**